doc/classes/KinematicBody2D.xml

```diff
@@ -80,6 +86,13 @@
 			<argument index="4" name="floor_max_angle" type="float" default="0.785398">
 			</argument>
 			<description>
+				Moves the body along the given vector. [code]linear_velocity[/code] should be given in pixels per second. If the body collides with another body, it will slide along the other body rather than stop immediately. Additionally, if the floor the body stands on is also a KinematicBody2D, the body will move together with the floor. This works regardless of the method used to move the floor, and is useful for making moving or rotating platforms in a platformer game.
+				[code]floor_normal[/code] is the up direction, used to determine what is a wall and what is a floor or a ceiling. If set to the default value of [code]Vector2(0, 0)[/code], everything is considered a wall. This is useful for topdown games.
+				When standing on a slope, the body will stop completely if the horizontal speed (relative to the floor's speed) goes below [code]slope_stop_min_velocity[/code]. If [code]slope_stop_min_velocity[/code] is set to 0, the body will never come to a complete rest on any slope.
+				If the body collides, it will change direction a maximum of [code]max_bounces[/code] times before it stops.
+				[code]floor_max_angle[/code] is the maximum angle (in radians) where a slope is still considered a floor (or a ceiling), rather than a wall.
+
+				Returns the movement that remained when the body stopped. To get more detailed information about collisions that occured, use [method get_slide_collision].
```

Comment:
Can you rephase it to be a bit simpler? Try to avoid the past + passive combo

Reply:
Sure. Which part specifically? Or do you mean all those sentences individually?